apps/web-giddh/src/app/shared/header/components/account-add-new-details/account-add-new-details.component.ts

```diff
@@ -462,17 +462,17 @@ export class AccountAddNewDetailsComponent implements OnInit, OnChanges, AfterVi
             } else {
                 // statesEle.setDisabledState(false);
                 if (this.isIndia) {
-                    statesEle.forceClearReactive.status = true;
-                    statesEle.clear();
-                    gstForm.get('stateCode').patchValue(null);
-                    gstForm.get('state').get('code').patchValue(null);
+                    // statesEle.forceClearReactive.status = true;
+                    // statesEle.clear();
+                    // gstForm.get('stateCode').patchValue(null);
+                    // gstForm.get('state').get('code').patchValue(null);
                 }
             }
         } else {
-                statesEle.forceClearReactive.status = true;
-                statesEle.clear();
-                gstForm.get('stateCode').patchValue(null);
-                gstForm.get('state').get('code').patchValue(null);
+                // statesEle.forceClearReactive.status = true;
```

Comment:
Should not be commented as it is used to reset the state after GST is reduced to empty value